lobsterpy/cohp/analyze.py

```diff
@@ -1605,7 +1605,7 @@ def get_lobster_calc_quality_summary(
 
             dos_lobster = doscar_lobster.completedos
 
-            vasprun = Vasprun(path_to_vasprun)
+            vasprun = Vasprun(path_to_vasprun, parse_potcar_file=False)

```

Comment:
This works with the latest pymatgen version, right?

Reply:
Yes, this change was necessary coz of latest pymatgen